plone/app/relationfield/tests.zcml

```diff
@@ -0,0 +1,16 @@
+<configure
```

Comment:
Hi @phgross,

thanks for your contribution and for writing proper tests for it! I have just one small improvement proposal. We try to avoid using test profiles in coredev tests because they make the tests harder to read (spread code across many different files) and also introduce global state into the test setup (those profile settings are available for all tests). Here is an example of how to test a behavior without a test profile: https://github.com/plone/plone.app.contenttypes/blob/master/plone/app/contenttypes/tests/test_behaviors_leadimage.py#L25

With this approach, you have the entire test setup in one file. It would be great if you could amend your code. 

Apart from that, the tests on jenkins seem to pass, so I guess we can merge after the refactoring.